qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -193,7 +193,10 @@ def retrieve_job(self, job_id):
             IBMQBackendError: if retrieval failed
         """
         try:
-            job_info = self._api.get_status_job(job_id)
+            job_info = self._api.get_job(job_id)
```

Comment:
This is a self-reminder that, in this PR, the endpoint called was changed as `get_status_job` does not support field inclusion yet, so the backend name is not provided. This is meant to be temporary - hopefully it will be implemented in the API soon, and then we can go back to using `get_status_job`, which is more efficient.